aldryn_newsblog/boilerplates/bootstrap3/templates/aldryn_newsblog/plugins/featured_articles.html

```diff
@@ -1,6 +1,6 @@
 {% load i18n %}
 
-<div class="aldryn aldryn-newsblog aldryn-newsblog-entries aldryn-newsblog-entries-featured">
+<div class="aldryn aldryn-newsblog aldryn-newsblog-articles aldryn-newsblog-articles-featured">
```

Comment:
we need to be careful with renaming there as these classes are already used on two sites...